config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -307,6 +307,88 @@ presubmits:
           securityContext:
             privileged: true
 
+  # manual test for pull-through cache registry used with 5k node tests
+  - name: pull-kubernetes-e2e-gce-pull-through-cache
+    cluster: k8s-infra-prow-build
+    always_run: false
+    skip_branches:
+      - release-\d+\.\d+ # per-release image
+    annotations:
+      fork-per-release: "false"
+      testgrid-dashboards: sig-k8s-infra-canaries
+      testgrid-tab-name: pull-kubernetes-e2e-gce-pull-through-cache
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+      preset-pull-kubernetes-e2e: "true"
+      preset-pull-kubernetes-e2e-gce: "true"
+    decorate: true
+    decoration_config:
+      timeout: 105m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: release
+      base_ref: master
+      path_alias: k8s.io/release
+    spec:
+      volumes:
+      - name: cache-secret
+        secret:
+          secretName: scale-pull-cache-token
+      containers:
+        - volumeMounts:
+          - name: cache-secret
+            readOnly: true
+            mountPath: /etc/registry-auth
+          env:
+          - name: KUBERNETES_REGISTRY_PULL_THROUGH_HOST
+            value: https://us-central1-docker.pkg.dev/v2/k8s-infra-e2e-scale-5k-project/k8s-5k-scale-cache/
+          - name: KUBERNETES_REGISTRY_PULL_THROUGH_BASIC_AUTH_TOKEN_PATH
+            value: /etc/registry-auth/token
```

Comment:
`token` comes from the key in the secret.

Reply:
yes. I'm not sure these env are winding up down through to configure-helper though, I don't see the job failing but I also don't see it pulling through any images. will have to look through ARTIFACTS when the run is complete